en/css/README.md

```diff
@@ -31,41 +31,44 @@ Looking nicer already!
 
 ## Static files in Django
 
-Another thing you will learn about today is called __static files__. Static files are all your CSS and images -- files that are not dynamic, so their content doesn't depend on request context and will be the same for every user.
+Finally we will take a closer look at these things we've been calling __static files__. Static files are all your CSS and images -- files that are not dynamic, so their content doesn't depend on request context and will be the same for every user.
 
-CSS is a static file, so in order to customize CSS, we need to first configure static files in Django. You'll only need to do it once. Let's start:
+CSS is a static file, so in order to customize CSS, we need to store them in a place that Django knows to look for static files.  
```

Comment:
A nit:  By saying it's "a static file" might give the impression CSS is usually just one file?  Maybe something like "CSS files are static files, so in order to customize CSS..."